cpu/stm32_common/periph/flashpage.c

```diff
@@ -41,46 +62,65 @@ void flashpage_write(int page, void *data)
 
     /* unlock the flash module */
     DEBUG("[flashpage] unlocking the flash module\n");
-    if (FLASH->CR & FLASH_CR_LOCK) {
-        FLASH->KEYR = FLASH_KEY1;
-        FLASH->KEYR = FLASH_KEY2;
+    if (CNTRL_REG & CNTRL_REG_LOCK) {
+        KEY_REG = FLASH_KEY1;
+        KEY_REG = FLASH_KEY2;
```

Comment:
You're redefining this values, but I don't see why you don't use them below, which makes me wonder what's the use of it after all.

Reply:
I'm sorry @kYc0o, but I'd like that you read this PR carefully before commenting (and also the datasheet). They are not reused below because they are not the same (FLASH->PEKEYR != FLASH->PRGKEYR).

Reply:
I know the datasheet and I know what is this about (my PR was there for about 6 months), I was referring to `CTRL_REG` and actually since you're defining `KEY_REG` you could also define `KEY_PREG` to be reused in other parts of the code, especially if your intention is to add the EEPROM driver.